rtrlib/rtr/packets.c

```diff
@@ -408,8 +440,9 @@ static bool rtr_pdu_check_size(const struct pdu_header *pdu)
 {
 	const enum pdu_type type = rtr_get_pdu_type(pdu);
 	const struct pdu_error *err_pdu = NULL;
+	const struct pdu_aspa *aspa_pdu = NULL;
 	bool retval = false;
-	uint64_t min_size = 0;
+	uint64_t expected_size = 0;
```

Comment:
`size_t`?

Reply:
Just a rule of thumb, it's good practice to use `size_t`every time you have sizes or memory addresses because `size_t` has the same size of pointers on this system `sizeof(size_t) == sizeof(void*)` 

(also in c++ `std::size_t`)

Reply:
"`size_t` can store the maximum size of a theoretically possible object of any type (including array)." (and it doesn't map to 32bit integer it maps to how large the address space is an this is on modern maschines 64bit)

`expected_size` expresses the length of the array therefore `size_t` is the perfect data type for it.